### ISSUE TITLE
ImportError: cannot import name 'ImageGPTFeatureExtractor' from 'transformers' (unknown location)

### ISSUE THREAD

**Issue by loukasilias** [huggingface/transformers#15396]
Hi all,

I am running the script in kaggle and face the error mentioned below:

```
----> 1 from transformers import ImageGPTFeatureExtractor, ImageGPTModel

ImportError: cannot import name 'ImageGPTFeatureExtractor' from 'transformers' (unknown location)

```
How could I fix this error?
Thank you!

**Comment by NielsRogge** [comment]
Hi,

What's the version of Transformers you have?

**Comment by loukasilias** [comment]
Hi,
4.5.1.
Do I need to upgrade ?

**Comment by LysandreJik** [comment]
ImageGPT was released in v4.13 I believe

**Comment by loukasilias** [comment]
Yes, you are right.
Thank you very much!
